quiz/quizfinal.js

```diff
@@ -28,7 +28,7 @@ const quizQuestions = {
         { answer: 5, correct: false },
```

Comment:
you were supposed to rename this function

Reply:
changed it, I guess I changed it back while doing command zs, will change it back